sdk/eventhub/azure-eventhubs/azure/eventhub/_common.py

```diff
@@ -81,105 +64,69 @@ def __init__(self, body=None):
         self.message.application_properties = {}
 
     def __str__(self):
-        dic = {
+        message_as_dict = {
```

Comment:
If there is any chance, even small that "body_as_str()" raise an exception, plesae try/except it. A `__str__` method must NOT raise ANY exception ever.